test/test_simulation.py

```diff
@@ -577,7 +577,66 @@ def test_ltf_eval(self):
                 random_instance=noise_prng_1,
             )
 
+            evaled_ltf_array = ltf_array.ltf_eval(transformed_inputs)
             assert_array_equal(
-                around(ltf_array.ltf_eval(transformed_inputs) + noise_prng_2.normal(size=(1, k)), decimals=10),
+                around(evaled_ltf_array + noise_prng_2.normal(loc=0, scale=1,
+                                                              size=shape(evaled_ltf_array)[0:2]), decimals=10),
```

Comment:
please update to `size=(N,k)` (cf my review comment to the corresponding implementation)